backend/docs/db/schema.sql

```diff
@@ -486,7 +491,10 @@ CREATE VIEW public.sequence_entries_view AS
         END AS processing_result
    FROM ((public.sequence_entries se
      LEFT JOIN public.sequence_entries_preprocessed_data sepd ON (((se.accession = sepd.accession) AND (se.version = sepd.version) AND (sepd.pipeline_version = ( SELECT current_processing_pipeline.version
-           FROM public.current_processing_pipeline)))))
+           FROM public.current_processing_pipeline
```

Comment:
Here again, this becomes rather unreadable - best to join the  public.current_processing_pipeline table